src/quo2/components/wallet/account_overview/view.cljs

```diff
@@ -0,0 +1,134 @@
+                                  (colors/custom-color customization-color 60 40)
+                                  (colors/custom-color customization-color 50 40)
+                                  theme))}]
+   [quo2/text
+    {:weight :medium
+     :size   :paragraph-2
+     :style  (style/currency-change (colors/theme-colors
+                                     (colors/custom-color customization-color 50)
+                                     (colors/custom-color customization-color 60)
+                                     theme))}
+    currency-change]
+   [icons/icon
+    (if up?
+      :i/positive
+      :i/negative)
+    {:color (colors/theme-colors (colors/custom-color customization-color 50)
+                                 (colors/custom-color customization-color 60)
+                                 theme)
+     :size  16}]])
+
+(defn- account-details [account-name account theme]
+  [rn/view {:style {:flex-direction :row}}
+   [quo2/text
+    {:weight :medium
+     :size   :paragraph-1
+     :style  (style/account-name (colors/theme-colors colors/neutral-100 colors/white theme))}
+    account-name]
+   (when (= account :watched-address)
+     [icons/icon :i/reveal
+      {:container-style style/reveal-icon
+       :color           (colors/theme-colors colors/neutral-80-opa-40
+                                             colors/white-opa-40
+                                             theme)}])])
+
+(defn- account-overview-internal
+  [{:keys [account time-frame time-frame-string time-frame-to-string account-name current-value
+           percentage-change currency-change theme metrics loading? customization-color]
+    :or   {customization-color :blue}}]
+  (let [time-frame-string (time-string time-frame time-frame-string)
+        up?               (= metrics :positive)]
+    [rn/view
+     {:style style/account-overview-wrapper}
+     (if loading?
+       [loading-state (colors/theme-colors colors/neutral-5 colors/neutral-90 theme)]
+       [:<>
+        [account-details account-name account theme]
+        [quo2/text
+         {:weight :semi-bold
+          :size   :heading-1
+          :style  style/current-value}
+         current-value]
+        [rn/view
+         {:style style/row-centered}
+         [:<>
+          [quo2/text
+           {:weight :medium
+            :size   :paragraph-2
+            :style  (style/bottom-time-text (and (not= :custom time-frame)
+                                                 (seq time-frame-to-string)))}
+           (when (seq time-frame-string)
+             time-frame-string)]
+          (when (and (= :custom time-frame)
+                     (seq time-frame-to-string))
+            [custom-time-frame time-frame-to-string])]
+         (when (and (seq percentage-change)
+                    (seq currency-change))
+           [numeric-changes percentage-change currency-change customization-color theme up?])]])]))
+
+(def view
+  "Create a account-overview UI component.
```

Comment:
Shouldn't the component be receiving a `state` parameter to match Figma? Why the exception for this Figma property if we're mirroring other properties?

I don't see a practical advantage to flatten the `state` property into boolean flags. The code is still simple enough if it uses the same names as Figma. It also helps developers create a preview screen that mirrors exactly what's in Figma (useful for designers too).